crates/hyperdrive-wrappers/.env.example

```diff
@@ -0,0 +1,2 @@
+HYPERDRIVE_REPO_URL=https://github.com/delvtech/hyperdrive.git
```

Comment:
is there going to be an install step to move this to `.env` or do we read directly from `.env.example`? If the latter, why not just call it `.env.`?